arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,242 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
 
-:::caution
+##### Storing your private key
 
-**_Never share your “secret recovery phrase” with anyone_**. Never enter it anywhere. A _private key_ is valid for an individual account only, but a _secret recovery phrase_ can be used to gain access to ALL accounts in your wallet.
+You might also want to add your developer's private key to your environment variables for easier and safer access.
+To do this, store your hex-encoded private key in a text file and add the path to the file as an ENV variable named `PRIVATE_KEY_PATH` to your shell` file .bashrc`, `.zshrc` or other. You can also add this variable to a `.env` file at the root of your project.
 
-:::
+#### Arbitrum Sepolia ETH
 
-### Testnet ETH
+Stylus settles to [Arbitrum Sepolia](./reference/testnet-information.md) testnet, so you will need at least one Sepolia ETH to deploy contracts and send transactions. 
 
-The Stylus testnet settles directly to the [Arbitrum Sepolia](/build-decentralized-apps/03-public-chains.md#arbitrum-sepolia) testnet. Follow these steps to acquire testnet ETH on the Stylus testnet:
+Start by visiting a faucet to obtain Sepolia testnet ETH. Here are a few options:
+    - [QuickNode Faucet for Arbitrum Sepolia](https://faucet.quicknode.com/arbitrum/sepolia)
+    - [Alchemy Sepolia Faucet](https://sepoliafaucet.com/)
+    - [Sepolia Faucet by PK910](https://sepolia-faucet.pk910.de/)
 
-1. Navigate to [https://bwarelabs.com/faucets/arbitrum-sepolia](https://bwarelabs.com/faucets/arbitrum-sepolia).
-2. Enter your wallet address into the text field.
-3. Click `Claim` and optionally follow the second step to receive extra testnet tokens.
-4. You should now have Sepolia ETH on the Stylus testnet.
+Now, your Sepolia ETH needs to be sent to Arbitrum Sepolia over the Arbitrum bridge:
 
-For additional sources of testnet ETH, please use a faucet on Arbitrum Sepolia or Ethereum Sepolia:
+1. Visit the [Arbitrum bridge](https://bridge.arbitrum.io/?destinationChain=arbitrum-sepolia&sourceChain=sepolia), Connect your wallet and authorize the bridge to access your tokens.
+2. Choose Arbitrum Sepolia as the source and Arbitrum Sepolia as the destination.
+3. Specify the amount of Sepolia ETH you want to bridge and initiate the transfer by clicking `Move funds to Arbitrum Sepolia`. Make sure to keep some ETH in the source chain to cover transaction fees!
 
-[https://faucet.quicknode.com/arbitrum/sepolia](https://faucet.quicknode.com/arbitrum/sepolia)
+To verify the transfer, you can track the transaction on the [Arbiscan block explorer](https://sepolia.arbiscan.io/).
 
-[https://sepoliafaucet.com/](https://sepoliafaucet.com/)
+## Creating a Stylus project with Cargo Stylus
 
-[https://sepolia-faucet.pk910.de/](https://sepolia-faucet.pk910.de/)
+`Cargo Stylus` is a CLI utility that helps you build, verify, and deploy Stylus contracts written in Rust.
```

Comment:
`cargo stylus` should be always lowercase. Btw, not sure if we should call it `cargo stylus` or `cargo-stylus` (although probably the former to avoid confusing devs with the command to use)

```suggestion
`cargo stylus` is a CLI utility that helps you check, build, verify, and deploy Stylus contracts written in Rust.
```

Reply:
This is still relevant. There are several mentions to `cargo Stylus`, and IMO should be `cargo stylus` or `cargo-stylus` (either one, but only one throughout the document).

Reply:
This was an oversight on my end. Should be fixed, now.